test/commands/scanner/rule/remove.test.ts

```diff
@@ -29,7 +29,11 @@ const customPathDescriptor = {
 
 const removeTest = setupCommandTest
 	.do(() => {
-		writeNewCustomPathFile();
+		writePopulatedCustomPathFile();
+	})
+	.finally(() => {
```

Comment:
While solving test failures in this class, I discovered that these tests don't properly clean up after themselves. So now they do.
Ideally, we'd be using stub, but I'm not sure I have the time to figure out how stubbing works in oclif tests.